tasks/internal_reconcile_report_orphan/src/adapters/storage/rdbms.py

```diff
@@ -23,7 +26,7 @@ def get_orphans_page(
             self,
             orphan_record_filter: OrphanRecordFilter,
             # todo: @bhazuka has expressed a desire to not pass loggers via parameters
-            logger: logging.Logger
+            logger: LOGGER
```

Comment:
You should not use an instantiated class as a typehint, and the typehint should still be logging.Logger. Revert changes to this file.

Reply:
reverted